scrapy/extensions/feedexport.py

```diff
@@ -93,12 +93,28 @@ def store(self, file):
 
 class S3FeedStorage(BlockingFeedStorage):
 
-    def __init__(self, uri):
+    def __init__(self, uri, access_key=None, secret_key=None):
+        # BEGIN Backwards compatibility for initialising without keys (and
```

Comment:
there is BEGIN, not no END. I think it is better to either remove this BEGIN from the comment, or end END comment.